src/components/countries/country/CountryContentCard/CountryContentCard.test.js

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable testing-library/no-container */
+/* eslint-disable testing-library/no-node-access */
+import { render } from '@testing-library/react';
+import React from 'react';
+import { CountryCardContent } from './CountryCardContent';
+
+describe('Test CountryCardContent component', () => {
+  it('should render CountryCardContent element by class', () => {
+    const { container } = render(<CountryCardContent />);
+    expect(container.getElementsByClassName('country-card-content')[0]).toBeInTheDocument();
+    expect(container.getElementsByClassName('country-card-content__country-block')[0]).toBeInTheDocument();

```

Comment:
Pass props to this component and verify whether correct data is rendered for each prop